src/Ch9/Ch9.Shared/Domain/SourceFeed.cs

```diff
@@ -7,17 +7,17 @@ namespace Ch9.Domain
     [Windows.UI.Xaml.Data.Bindable]
     public class SourceFeed
     {
-        public SourceFeed(string url, string name = null, Uri image = null)
+        public SourceFeed(string feedUrl, string name = null, Uri image = null)
         {
-            Url = url;
+            FeedUrl = feedUrl;
             Name = name;
-            Image = image != null ? image : new Uri("https://channel9.msdn.com/assets/images/nineguy-512-bw.png");
+            ThumbnailUrl = image != null ? image : new Uri("https://channel9.msdn.com/assets/images/nineguy-512-bw.png");
```

Comment:
You could rename image to match the public property and use the `??` operator instead of `? : `